docs/witx/typenames.witx

```diff
@@ -461,7 +461,7 @@
 (typename $oflags
   (flags (@witx repr u16)
     ;;; Create file if it does not exist.
-    $creat
```

Comment:
This should be ignored:
https://github.com/WebAssembly/WASI/blob/main/phases/snapshot/witx/typenames.witx#L443-L455